packages/salesforcedx-apex-debugger/src/adapter/apexDebug.ts

```diff
@@ -1034,6 +1039,23 @@ export class ApexDebug extends LoggingDebugSession {
           typerefs: Array.from(exceptionBreakpoints.keys())
         };
         break;
+      case TERMINATE_SESSION_REQUEST:
+        this.sendEvent(new TerminatedEvent());
```

Comment:
A request from the extension to the adapter to terminate the session because the user responded with `No` to the question `Are you still debugging?`